app/dependency.rb

```diff
@@ -28,6 +30,23 @@ def github_repo_url
     Github.client.web_endpoint + github_repo
   end
 
+  def github_compare_url
+    return unless github_repo
+    @compare_url ||= begin
```

Comment:
Could we refactor the `begin` block into another method, as in `#changelog_url`?